hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/SessionRepository.java

```diff
@@ -66,6 +66,12 @@ public SessionId createSession(SessionStartedState startedState) {
         throw new InvalidSessionStateException(sessionId, expectedStateClass, currentState.getClass());
     }
 
+    @Timed(name = Urls.SESSION_REPO_TIMED_GROUP)
+    public StateController getUnknownStateController(final SessionId sessionId) {
```

Comment:
I think perhaps either `getFallbackStateController` or alternatively `getStateControllerRegardlessOfCurrentState`...

But it might be worth considering dropping this method, and just having `getStateController` allow `null` to be passed in as `expectedStateClass`. Your call.

Reply:
The latter proved quite hard, so I went for the former.